utils/sysfs/fakesysfs/fake.go

```diff
@@ -53,6 +54,55 @@ func (self *FileInfo) Sys() interface{} {
 type FakeSysFs struct {
 	info  FileInfo
 	cache sysfs.CacheInfo
+
+	nodesPaths  []string
+	nodePathErr error
+
+	cpusPaths  map[string][]string
+	cpuPathErr error
+
+	coreThread map[string][]byte
+	coreIDErr  error
+
+	memTotal []byte
+	memErr   error
+
+	hugePages    []os.FileInfo
+	hugePagesErr error
+
+	hugePagesNr    map[string][]byte
+	hugePagesNrErr error
+}
+
+func (self *FakeSysFs) GetNodesPaths() ([]string, error) {
+	if len(self.nodesPaths) == 0 {
```

Comment:
What is a purpose of this conditional statement? It could be clearer if you just used `return self.nodesPaths, self.nodePathErr` but set both properties to expected values in test code.